source/features/useful-not-found-page.tsx

```diff
@@ -223,6 +249,15 @@ void features.add(import.meta.url, 	{
 		pageDetect.isPRCommit404,
 	],
 	init: onetime(initPRCommit),
+}, {
+	asLongAs: [
+		pageDetect.isRepoFile404,
+	],
+	include: [
+		pageDetect.isRepoTree,
+		pageDetect.isSingleFile,
+	],
+	init: onetime(initRepoFile),
```

Comment:
```suggestion
	init: initRepoFile,
```

The observers are disabled on page unload via the AbortSignal, so we can't and don't need to use `onetime`